edx_proctoring/templates/proctored_exam/failed-prerequisites.html

```diff
@@ -26,23 +26,26 @@
     {% endfor %}
     </ol>
   </p>
-  <p>
+
+  {% if allow_unproctored %}
+    <p>
     {% blocktrans %}
     Due to unsatisfied prerequisites, you can only take this exam without proctoring.
     {% endblocktrans %}
-  </p>
+    </p>
 
-  <button class="gated-sequence start-timed-exam" data-attempt-proctored=false>
+    <button class="gated-sequence start-timed-exam" data-attempt-proctored=false>
```

Comment:
@robrap thanks for the suggestion. I've made a shared template and used it in three of the four places. Let me know if it is what you had in mind.